substratevm/src/com.oracle.svm.jni/src/com/oracle/svm/jni/access/JNIAccessFeature.java

```diff
@@ -126,27 +129,36 @@ public void afterRegistration(AfterRegistrationAccess arg) {
                         ConfigurationFiles.Options.JNIConfigurationFiles, ConfigurationFiles.Options.JNIConfigurationResources, ConfigurationFile.JNI.getFileName());
     }
 
-    private class JNIRuntimeAccessibilitySupportImpl implements JNIRuntimeAccess.JNIRuntimeAccessibilitySupport, ReflectionRegistry {
+    private class JNIRuntimeAccessibilitySupportImpl extends PredicatedConfigurationSupport implements JNIRuntimeAccess.JNIRuntimeAccessibilitySupport, ReflectionRegistry {
```

Comment:
I can't find class `PredicatedConfigurationSupport` in this changeset.

Reply:
Added. I for got to add it after fixing comments from @christianwimmer.